pymepix/processing/logic/packet_processor.py

```diff
@@ -0,0 +1,302 @@
+from enum import IntEnum
```

Comment:
Not sure if it would have been better because this is actually a new class that provides the functionality of the previous packet processor with the same name. The only extracted functionality that is not provided is the inter process communication which is performed in the `pipeline_packet_processor` now.

Header has been added now.